packages/urls/src/urls.ts

```diff
@@ -1,167 +1,296 @@
+);
+export const HELP_CENTER_KEEPING_SEED_SAFE_URL: Url = withPlatformUtm(
+    'https://trezor.io/learn/basics/keeping-your-wallet-backup-safe',
+);
+export const HELP_CENTER_TRANSACTION_FEES_URL: Url = withPlatformUtm(
+    'https://trezor.io/guides/trezor-suite/trezor-suite-desktop/transaction-fees-in-trezor-suite',
+);
+export const HELP_CENTER_EVM_ADDRESS_CHECKSUM: Url = withPlatformUtm(
+    'https://trezor.io/guides/trezor-suite/trezor-suite-desktop/experimental-features-in-trezor-suite',
+);
+export const HELP_CENTER_EVM_SEND_TO_CONTRACT_URL = withPlatformUtm(
+    'https://trezor.io/support/troubleshooting/coins-tokens/where-is-my-ethereum',
+);
+export const HELP_CENTER_FIRMWARE_REVISION_CHECK: Url = withPlatformUtm(
+    'https://trezor.io/learn/security-privacy/how-trezor-keeps-you-safe/trezor-firmware-revision-check',
+);
+export const HELP_CENTER_FIRMWARE_REVISION_CHECK_MOBILE: Url = withPlatformUtm(
+    'https://trezor.io/guides/trezor-suite/trezor-suite-lite/trezor-firmware-authenticity-check-on-mobile',
+);
+export const HELP_CENTER_ENTROPY_CHECK_URL: Url = withPlatformUtm(
+    'https://trezor.io/learn/security-privacy/how-trezor-keeps-you-safe/entropy-check',
+);
+
+export const HELP_CENTER_REPLACE_BY_FEE_ETHEREUM: Url = withPlatformUtm(
+    'https://trezor.io/guides/trezor-suite/trezor-suite-desktop/replace-by-fee-rbf-ethereum',
+);
+export const HELP_CENTER_REPLACE_BY_FEE_BITCOIN = withPlatformUtm(
+    'https://trezor.io/guides/trezor-suite/trezor-suite-desktop/replace-by-fee-rbf-bitcoin',
+);
+export const HELP_CENTER_CANCEL_TRANSACTION: Url = withPlatformUtm(
+    'https://trezor.io/support/troubleshooting/trezor-suite-issues/can-i-cancel-or-reverse-a-transaction',
+);
 // TODO: update this link when the article is ready
-export const HELP_CENTER_SOL_SEND: Url =
-    'https://trezor.io/learn/supported-assets/solana/solana-sol-on-trezor';
+export const HELP_CENTER_SOL_SEND: Url = withPlatformUtm(
+    'https://trezor.io/learn/supported-assets/solana/solana-sol-on-trezor',
+);
 // TODO: update this link when the article is ready
-export const HELP_CENTER_SOLANA_HELP_URL: Url =
-    'https://trezor.io/support/troubleshooting/coins-tokens/where-is-my-solana';
+export const HELP_CENTER_SOLANA_HELP_URL: Url = withPlatformUtm(
+    'https://trezor.io/support/troubleshooting/coins-tokens/where-is-my-solana',
+);
+
+export const UNINSTALL_BRIDGE_URL: Url = withPlatformUtm(
+    'https://trezor.io/guides/trezor-suite/trezor-suite-desktop/deprecation-and-removal-of-standalone-trezor-bridge',
+);
+
+export const EXPERIMENTAL_FEATURES_KB_URL: Url = withPlatformUtm(
+    'https://trezor.io/learn/a/experimental-features-in-trezor-suite',
+);
+
+// =====================
+// 🛠️ SERVICES
+// =====================
+
+export const GEOLOCATION_API_URL = withPlatformUtm('https://services.trezor.io/get-country/');
+export const IMAGE_PROXY_API_URL = withPlatformUtm('https://services.trezor.io/image-proxy/');
```

Comment:
I don't think it makes sense to add utm_medium for those two services. Please remove it 🙏